testing/test_many.sh

```diff
@@ -3,7 +3,8 @@
 source testing/test_preamble.sh
 
 NUM_DEVICES=8
-
+RUN_LIMIT=10
```

Comment:
With this change, can the run limit be increased?   I'm basically wondering if this change (dhcp timeout) has a measurable impact on the overall time to run the ~10 runs.